frontend/src/pages/dashboard/shared/forms/HRD.tsx

```
@@ -19,10 +19,35 @@ export const HRD: React.FC<HRDProps> = ({
 
     if (!entityType) return null
 
-    const staffOptions = employees?.map((emp: any) => ({
-        value: emp.id,
-        label: emp.staffName || emp.employeeName || emp.name || `Staff ${emp.id}`
-    })) || []
+    const staffOptions = React.useMemo(() => {
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)